tasks/test_results_processor.py

```diff
@@ -334,7 +334,10 @@ def create_daily_total():
         # Upsert Tests
         if len(test_data) > 0:
             test_insert = insert(Test.__table__).values(
-                sorted(test_data.values(), key=lambda x: str(x["flags_hash"]))
+                sorted(
+                    test_data.values(),
+                    key=lambda x: str(x["test_id"]),
```

Comment:
Getting 3 errors w/ this change, this intentional?

Reply:
my latest change should fix those

Reply:
Sweet!